nerfstudio/utils/math.py

```diff
@@ -288,24 +288,24 @@ def safe_normalize(
     return vectors / (torch.norm(vectors, dim=-1, keepdim=True) + eps)
 
 
-def masked_reduction(loss: TensorType[0], M: TensorType[0], reduction_type: Literal["image", "batch"]):
+def masked_reduction(input_tensor: TensorType[0], mask: TensorType[0], reduction_type: Literal["image", "batch"]):
```

Comment:
I missed this earlier, I think the size is incorrect here, `TensorType[0]` is a single number.